src/witan/send/model.clj

```diff
@@ -23,7 +22,7 @@
 
    ;;Loop
    [:select-starting-population :apply-state-changes]
-   [:get-transition-matrix :apply-state-changes]
+   [:adjust-joiners-transition :apply-state-changes]
```

Comment:
So these changes haven't yet been applied to the flowchart? Keen to close the branch sooner rather than later, so if that's the case, let's just make that a new story/branch